docs/api-docs/storefront/graphql/graphql-api-overview.mdx

```diff
@@ -194,60 +189,76 @@ Content-Type: application/json
 </Tab>
 </Tabs>
 
-Customer impersonation token-authenticated requests made to the GraphQL Storefront API receive store information from the perspective of the customer corresponding to the customer ID specified in the `X-Bc-Customer-Id` header sent with the GraphQL `POST` request. Pricing, product availability, customer account, and customer details will be reflected.
+Customer impersonation token-authenticated requests made to the GraphQL Storefront API may query store information from the perspective of ANY customer by specifying the customer ID using the `x-bc-customer-id` header sent with the GraphQL `POST` request. Customer-specific data such as product pricing & availability, customer account details, and other parameters will change to reflect the specified customer's data.
+
+Unlike typical GraphQL Storefront API tokens, Customer impersonation tokens _are_ sensitive, and should therefore **never** be exposed publicly, for example, to JavaScript or HTML. These tokens should not be used for front-end requests. They should be treated with the same care as other application secrets, just as you might treat an OAuth access token for BigCommerce's REST APIs. Requests using these tokens originating from a web browser will be rejected.
 
-Customer impersonation tokens should **never** be exposed publicly, for example, to JavaScript or HTML. These tokens should not be used for front-end requests. Unlike normal GraphQL Storefront API tokens, they are sensitive and should be treated like secrets, just as you might treat an OAuth access token for BigCommerce's REST APIs. Attempts to run requests using these tokens from a web browser will be rejected.
+It is not necessary to generate a new token for each customer ID. You may use a single token at any given time for your application, and specify the customer ID on a per-request basis.
```

Comment:
```suggestion
It is optional to generate a new token for each customer ID. You may use a single token at any given time for your application and specify the customer ID per request.
```

Reply:
"optional" implies that you might want to do it sometimes, which is different from my intent. My intent is to discourage doing this, because we've noticed folks are thinking it's how the tokens are meant to work when it's actually an antipattern. I'll adjust.

Reply:
ok, thanks